README/Tactic/Cong.agda

```diff
@@ -0,0 +1,67 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Examples of how to use Tactic.Cong
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K #-}
+
+module README.Tactic.Cong where
+
+-- Tactic.Cong provides a helper macro for using congruence when reasoning about
+-- relations. It allows the "hole" that you are operating in to be inferred from
+-- the relation instance you are operating on, through use of ⌊_⌋.
+
+-- Tactic.Cong is inspired by the agda-holes project.
+
+-- For this README, the relation used is _≡_, though any (non-dependent) relation
+-- with congruence and transitivity defined should work
```

Comment:
This is great news, any chance we could add an example as a test?

Does it work for non-concrete relations, e.g. ones provided as arguments?

Reply:
Yeah that would be a good idea.

It's intended to work with relations defined directly as an argument to the module, however I've just thought of an edge case that I did not cover. I'll fix that.

Reply:
Hmm, to work with any arbitrary term for the relation, I think it actually would be necessary to have a user-supplied function for extracting the left and right hand sides of the relation from a given instance of it. Instead, what I have at the moment should cover a lot of common use cases, but not all. A serviceable workaround for those remaining cases is for the user to create a definition then use that as the argument.

Perhaps it would be best to leave improving this to a future PR?

Reply:
There is now an example using `≤` for `ℤ`.

I don't think the unsupported forms of inline relation definitions are that big of a deal. At any rate I think they can eventually be supported in a backward compatible way, so I'll leave that out of this PR.

Reply:
Actually, I think I might be able to make `_≈⌊_⌋_` robustly work with inline relations with no restrictions, but not the standalone macro `⌊⌋`.

Reply:
Implemented. Resolved.